vm_manager/vm_manager_cluster.py

```diff
@@ -155,7 +155,7 @@ def _configure_vm( vm_options ):
     logger.info("libvirt xml config defined for VM " + vm_options["name"])
 
     # Enable on Pacemaker
-    if "disable" not in vm_options or ("disable" in vm_options and not vm_options["disable"]):
+    if "enable" not in vm_options or ("enable" in vm_options and vm_options["enable"]):
```

Comment:
I don't understand the first part `if "enable" not in vm_options`.
Here we want to check if enable is pass to true as an argument, so I propose
`if "enable" in vm_options and vm_options["enable"]`

Reply:
If you write it like that then the default becomes "disable" (if the enable key is not present, then we don't enable the guest).
I felt the default should be "enable", so I added this condition : if "enable" is not in the dict, then we enable the guest.

Reply:
Okay, seems right to me